tests/setup-files/setup.js

```diff
@@ -0,0 +1,3 @@
+import path from 'path'
+
+global.SETUP_JS_MODULE = path.basename(import.meta.url)
```

Comment:
`import.meta.url` is an url, calling basename on it might be not ideal
does just `import.meta.filename` work here?

Reply:
ah, this is actually testing that `import path` works, makes sense

Reply:
yeah, these 3 setup files are just for testing.